interfaces/Portalicious/src/app/services/auth/strategies/msal-auth/msal-auth.callback.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="flex h-screen w-full flex-col items-center justify-center">
+  <p-progressSpinner />
+  <h2 i18n>Redirecting to 121&hellip;</h2>
+</div>
```

Comment:
I think this should be in the `AuthCallbackPageComponent`, and the `MsalAuthCallbackComponent` component should actually just have empty HTML (you could use `template: ''` instead of `templateUrl`). With the idea that the auth-callback page will always look the same regardless of the strategy.

EDIT: it makes me wonder whether the callback-component is needed at all, or whether the strategy should just implement a "authCallback" function, which in the case of msal-auth would just contain the contents of the constructor below.